### PR TITLE
chore: add title example to the demo

### PR DESCRIPTION
**Pull request by splincode**
## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring
- [ ] Build or CI related changes
- [ ] Documentation content changes

## What is the current behavior?

image

## What is the new behavior?

image

image

### CODE REVIEW

File: projects/kit/components/input-files/input-files.template.html [hunk 1]
```diff
@@ -31,6 +31,7 @@
             <input
                 #input
                 type="file"
+                title=""
```
Comment (nsbarsukov) [inline projects/kit/components/input-files/input-files.template.html:34]: `options.title` ?

**Review by nsbarsukov — APPROVED** [review]

File: projects/kit/components/input-files/input-files.template.html (hunk 1 above)
Comment (splincode) [reply projects/kit/components/input-files/input-files.template.html:34]: @waterplea what do you think?

Comment (splincode) [reply projects/kit/components/input-files/input-files.template.html]: fixed

**Review by waterplea — COMMENTED** [review]
Our stance is to use native `input` if you need native attributes. Let's not add it to the options.

**Comment by waterplea** [comment]
Maybe just hardcode empty line for the built-in input?

**Comment by splincode** [comment]
@waterplea fixed

File: projects/kit/components/input-files/input-files.directive.ts [hunk 2]
```diff
@@ -27,22 +27,27 @@ export class TuiInputFilesDirective {
...
     }
 
     @HostBinding('multiple')
     get multiple(): boolean {
-        return this.el.nativeElement.multiple ?? this.options.multiple;
+        return this.input.multiple ?? this.options.multiple;
+    }
+
+    @HostBinding('title')
+    get title(): string | null {
```
Comment (waterplea) [inline projects/kit/components/input-files/input-files.directive.ts:44]: Is this necessary?

**Review by waterplea — APPROVED** [review]
I guess this should be called `chore: add title example to the demo` now )

File: projects/kit/components/input-files/input-files.directive.ts (hunk 2 above)
Comment (splincode) [reply projects/kit/components/input-files/input-files.directive.ts]: fixed

**Pull request merged**